docs/infrastructure.md

```diff
@@ -38,7 +38,7 @@ The `utils` folder contains a number of useful utility functions, for use in var
 
 ### API utilities
 
-`loadConfig()` loads in the `config.json` file (by fetching it at runtime, not as an imported module) in order to gain access to the API location, version number etc. It returns a promise, which when resolved signals the availability the API. This is required to be called first and allowed to complete before dispatching API actions using the `buildUrl()` helper function. <!-- Edit this to generalize if/when we have more API helper funcs  --> It also sets `window.orcVersion` to the current Orckestra Commerce Suite version number.
+`loadConfig()` loads in the `config.json` file (by fetching it at runtime, not as an imported module) in order to gain access to the API location, version number etc. It returns a promise, which when resolved signals the availability the API. This is required to be called first and allowed to complete before dispatching API actions using the `buildUrl()` helper function. <!-- Edit this to generalize if/when we have more API helper funcs
 
```

Comment:
Yikes, you have accidentally commented out half the doc file here!

Reply:
Yikes indeed, sorry for the misstep.